cylc/flow/cfgspec/workflow.py

```diff
@@ -1730,57 +1730,34 @@ def get_script_common_text(this: str, example: Optional[str] = None):
                 ''')
 
             with Conf('workflow state polling', desc=f'''
-                Configure automatic workflow polling tasks as described in
-                :ref:`WorkflowStatePolling`.
+                Deprecated support for automatic workflow state polling tasks
+                as described in :ref:`WorkflowStatePolling`. Note the Cylc 7
+                "user" and "host" config items are not supported.
 
-                The items in this section reflect
-                options and defaults of the ``cylc workflow-state`` command,
-                except that the target workflow ID and the
-                ``--task``, ``--cycle``, and ``--status`` options are
-                taken from the graph notation.
-
-                .. versionchanged:: 8.0.0
+                .. versionchanged:: 8.3.0
```

Comment:
This was changed back in 8.0.0

```suggestion
                .. versionchanged:: 8.0.0
```